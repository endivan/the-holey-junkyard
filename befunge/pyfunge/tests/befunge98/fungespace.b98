# va"derongi v :DAB">:#,_@_,#! #:<"BAD: v reflects"a
  <@_,#! #:<"BAD: cells past 80 columns are ignored"                            v
 >a"derongi era senalp tneuqesbus :DAB">:#,_@
< @_,#! #:<"BAD: second plane overwrites first plane"a0
 v                                                                              <
 '
 vv
v>:ac*2--#;_0a"derongi si deefmrof dna detanetacnoc era senalp tneuqesbus :DOOG">:#,_@,a.$_,#! #:<"BAD: Funge space contains unexpected character "0;
 
