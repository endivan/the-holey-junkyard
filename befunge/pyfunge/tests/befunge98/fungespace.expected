GOOD: subsequent planes are concatenated and formfeed is ignored

